src/components/SignOut/SignOut.tsx

```
@@ -1,85 +1,28 @@
- * - `@apollo/client`: For GraphQL mutation handling.
- * - `react-router-dom`: For navigation after logout.
- *
- * @cssModule
- * - `style/app-fixed.module.css`: Contains styles for the sign-out button and container.
- */
-import React from 'react';
-import styles from 'style/app-fixed.module.css';
+import styles from '../../style/app-fixed.module.css';
 import LogoutIcon from '@mui/icons-material/Logout';
 import useSession from 'utils/useSession';
 import { REVOKE_REFRESH_TOKEN } from 'GraphQl/Mutations/mutations';
```

ESLint

[error] 'REVOKE_REFRESH_TOKEN' is defined but never used.

(@typescript-eslint/no-unused-vars)